Stripe/UI/UIImage+Stripe.m

```diff
@@ -0,0 +1,44 @@
+//
+//  UIImage+Stripe.m
+//  Stripe
+//
+//  Created by Ben Guo on 1/4/16.
+//  Copyright © 2016 Stripe, Inc. All rights reserved.
+//
+
+#import "UIImage+Stripe.h"
+#import "STPPaymentCardTextFieldViewModel.h"
+
+@implementation UIImage (Stripe)
+
++ (UIImage *)stp_amexCardImage {
```

Comment:
ah, ok – I misread your "all we really want here" and reduced the scope of the changes. This is actually pretty close to what I had in https://github.com/stripe/stripe-ios/pull/288 .